### ISSUE TITLE
[Test Failure][Accessibility][RUS] Several drop down boxes and strings are truncated in the right-panel of PM UI

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#4519]
**NuGet Version:** Dev 2265 VS15 Localized NuGet Client Vsix
**VS Version:** D15Rel 26205.00 (select all languages and workloads)
**OS Version:** Windows 10 RS1_Release 14393.821.170127-1801

**Repro Steps:**
1.  Select **RUS** through ‘Tools -> Options -> International Settings’.
2.  Restart VS, and create a new C# Class Library (.NET Framework) via VS.
3.  Right-click on the project and click ‘Manage NuGet Packages…’.
4.  Click ‘Browse’ tab and select one package and click ‘Options’ drop down button in the right-panel of PM UI.
5.  Check the **‘Dependency behavior:’** and **‘File conflict action:’** drop down boxes.

**Expected Result:**
The ‘Dependency behavior:’ and ‘File conflict action:’ drop down boxes show correctly.

**Actual Result:**
The ‘Dependency behavior:’ and ‘File conflict action:’ drop down boxes are truncated as below screenshot. String: **‘Show preview window’, ‘Force uninstall, even if there are dependencies on it’** and hyper-link **‘Learn about Install Options’** are truncated too. 

The problem is these strings are not auto-wrapped and there is no horizontal scroll bar on the bottom of the right-panel.
![rus](https://cloud.githubusercontent.com/assets/20632625/22686303/8be44ec6-ed5e-11e6-8f8b-6c51e185c0f7.png)

**Comment by rrelyea** [comment]
@mishra14  - please open an internal bug, we'll see if we can get an exception for now.

**Comment by mishra14** [comment]
@rrelyea this is an accessibility error, I will change the title and create an appropriate bug.

**Comment by rrelyea** [comment]
Please refer to the bug # in this issue.

**Comment by mishra14** [comment]
Internal bug: 382204

**Comment by mishra14** [comment]
This is observed in dev14 as well - 
![capture](https://cloud.githubusercontent.com/assets/10507120/22994229/0212d9f6-f37b-11e6-98f2-fba6b1d3f621.PNG)

**Comment by mishra14** [comment]
@rrelyea: This repros on dev14 and dev15[English] as well. This is an accessibility issue as the string truncation happens only if the width/size of the options tab is reduced.

**Comment by v-luzh** [comment]
@mishra14 This issue repro with the default window layout. This also repro on OS: Win10 RTM RUS+VS 2015 U3 RUS with the default window layout as below screenshot.
![4519reproondev14](https://cloud.githubusercontent.com/assets/20632625/23011368/9ddf48ca-f45b-11e6-8558-7bcf2c7589cc.png)
